pages/linux/systemd-sysusers.md

```diff
@@ -0,0 +1,16 @@
+# systemd-sysusers
+
+> Create system users and groups.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-sysusers.html>.
+
+- Create users and groups from a specific configuration file:
+
+`systemd-sysusers {{config_pathpath}}`
+
+- Simulate processing a configuration file without creating users and groups to preview the entries that would be generated:
+
+`systemd-sysusers --dry-run {{config_pathpath}}`
+
+- Print the contents of the specified configuration file:
+
+`systemd-sysusers --cat-config {{config_path}}`
```

Comment:
This doesn't work, the path is ignored.
If you want to do this, you can just use `cat` instead.

```suggestion
- Print the contents of all config files (before each file, its name is printed as a comment):

`systemd-sysusers --cat-config`
```